README.md

```diff
@@ -326,17 +329,13 @@ Currently for AOSP browsers 4.0.x - 4.1.x, the panes do not animate smoothly whe
 
 ### Requirements
 
-* [Node.js 0.10.x/npm](http://nodejs.org/download/)
+* [Node.js v4.x LTS + NPM v2.x](https://nodejs.org/en/download/) (Mobify recommends [NVM](https://github.com/creationix/nvm) for installing Node + NPM)
```

Comment:
So this is saying Node 4.x and NPM 2.x are required? Couple questions:
1. Is this correct? I've tried it on Node 5 + NPM 3 and it works. 
2. There's been some confusion with people thinking this is "minimum requirements" not "exact requirements". If it's correct, perhaps we could include minimum requirements and mention max requirements separately so clarify (eg. min: 4.0, max: 4.2)